playbooks/common/openshift-cluster/config.yml

```diff
@@ -31,6 +31,6 @@
 
 - include: ../openshift-master/config.yml
 
-- include: additional_config.yml
-
 - include: ../openshift-node/config.yml
+
+- include: additional_config.yml
```

Comment:
I'm not sure I'm a fan of this...  having additional config happen before nodes was kind of nice.